cpp/src/svm/linear.cu

```diff
@@ -463,7 +351,7 @@ LinearSVMModel<T> LinearSVMModel<T>::fit(const raft::handle_t& handle,
       parallel ? WorkerHandle(handle, omp_get_thread_num()) : WorkerHandle(handle, stream);
     if (nClasses > 1) {
       raft::linalg::unaryOp(
-        yi, y, nRows, OvrSelector<T>{model.classes, nClasses == 2 ? 1 : class_i}, worker.stream);
+        yi, y, nRows, OvrSelector<T>{classes, nClasses == 2 ? 1 : class_i}, worker.stream);
     }
     T target;
     int num_iters;
```

Comment:
It is not visible in the PR, but it looks like there is some dead code to remove. These two vectors are left unused.
```
std::vector<T> targets(coefCols);
std::vector<int> num_iters(coefCols);
```